src/lnn.jl

```diff
@@ -0,0 +1,42 @@
+"""
+Constructs a Lagrangian Neural Network [1].
+
+References:
+[1] Miles Cranmer, Sam Greydanus, Stephan Hoyer, Peter Battaglia, David Spergel, and Shirley Ho.Lagrangian Neural Networks.
+    InICLR 2020 Workshop on Integration of Deep Neural Modelsand Differential Equations, 2020.
+"""
+
+struct LagrangianNN
```

Comment:
You might want to specify the types as done in https://github.com/SciML/DiffEqFlux.jl/blob/16ac36f98a4c361fb5c81d67414a057214694a96/src/hnn.jl#L35

Reply:
Thanks, I`ll do it